### PR TITLE
update label of icon

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
## 🛠️ Fixes Issue
The icon label is not proper.

## 👨‍💻 Changes proposed
- Update icon of label.
- fix grammer probs

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

File: pages/How-to/guide/starting-discussion.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+## Starting Discussion in GitHub
+1. Go to the **repo** in which you want to start a discussion in or use this link - `https://github.com/{repo-owner}/{repo-name}/discussions`.
+2. Beside the **Pull request or Issues icon**, below the repo name, if you see the "Discussions" tap means, that the repo accepts discussions, if you don't see the tab, the repo doesn't accept discussions... If you used the link mentioned above, and it says 404 Page Not Found, means that the repo doesn't accept discussions.
+   ![image](https://user-images.githubusercontent.com/84712013/169663113-99dd25e7-c3ad-4556-94a4-39d8f8caa992.png)
+3. If you see the tab near the Pull request or Issues or if the link above, worked, and you want to start a discussion, tap the "New Discussion" icon.
```
Comment (vikasganiga05) [inline pages/How-to/guide/starting-discussion.md:5]:
```suggestion
3. If you see the tab near the Pull request or Issues or if the link above worked and want to start a discussion, tap the "New Discussion" icon.
```

File: pages/How-to/guide/starting-discussion.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+4. Select the **CATERGORY** of your discussion.
+   ![image](https://user-images.githubusercontent.com/84712013/169663148-c967bb82-0816-4689-94db-56ae05b58666.png)
+5. Add title and description of the discussion you want to start.
```
Comment (vikasganiga05) [inline pages/How-to/guide/starting-discussion.md:8]:
```suggestion
5. Add a title and description of the discussion you want to start.
```

File: pages/How-to/guide/starting-discussion.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+   ![image](https://user-images.githubusercontent.com/84712013/169663162-447b0bc6-0178-4e72-adbe-8c9e5333fc6a.png)
+6. Tap "**Start Discussion"** button to open your discussion..
```
Comment (vikasganiga05) [inline pages/How-to/guide/starting-discussion.md:10]:
```suggestion
6. Tap the "**Start Discussion"** button to open your discussion...
```

Here are some suggestions from [Grammarly](https://www.grammarly.com/).

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
I scanned this doc through [Grammarly](https://www.grammarly.com/) and it suggested the following changes. I don't know if it's accurate or not. Just a suggestion.

File: pages/How-to/guide/starting-discussion.md [hunk 4]
```diff
@@ -1,4 +1,17 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+## Starting Discussion in GitHub
```
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply pages/How-to/guide/starting-discussion.md:1]: this one is my bad

File: pages/How-to/guide/starting-discussion.md [hunk 5]
```diff
@@ -1,4 +1,17 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+## Starting Discussion on GitHub
+1. Go to the **repo** in which you want to start a discussion or use this link - `https://github.com/{repo-owner}/{repo-name}/discussions`.
+2. Besides the **Pull request or Issues icon**, below the repo name, if you see the "Discussions" tap means, that the repo accepts discussions, if you don't see the tab, the repo doesn't accept discussions... If you used the link mentioned above, and it says 404 Page Not Found, means that the repo doesn't accept discussions.
```
Comment (KendallDoesCoding) [inline pages/How-to/guide/starting-discussion.md:3]:
```suggestion
2. Besides, the **Pull request or Issues icon**, below the repo name, if you see the "Discussions" tap means, that the repo accepts discussions, if you don't see the tab, the repo doesn't accept discussions... If you used the link mentioned above, and it says 404 Page Not Found, means that the repo doesn't accept discussions.
```

**Comment by KendallDoesCoding** [comment]
How to resolve conflicts?

**Comment by vikasganiga05** [comment]
> 



> How to resolve conflicts?

He @KendallDoesCoding , The following resources may help you.
- GitHub Docs &rarr; [Resolving a merge conflict on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
- YouTube Video &rarr; [1.9: Resolving Merge Conflicts - Git and GitHub for Poets](https://youtu.be/JtIX3HJKwfo)

**Comment by KendallDoesCoding** [comment]
> > 
> 
> > How to resolve conflicts?
> 
> He @KendallDoesCoding , The following resources may help you.
> 
> * GitHub Docs → [Resolving a merge conflict on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> * YouTube Video → [1.9: Resolving Merge Conflicts - Git and GitHub for Poets](https://youtu.be/JtIX3HJKwfo)

Hi Vikas, Thanks I know how to resolve but it's a little complex for this one

**Comment by vikasganiga05** [comment]
> > > 
> > 
> > 
> > > How to resolve conflicts?
> > 
> > 
> > He @KendallDoesCoding , The following resources may help you.
> > 
> > * GitHub Docs → [Resolving a merge conflict on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> > * YouTube Video → [1.9: Resolving Merge Conflicts - Git and GitHub for Poets](https://youtu.be/JtIX3HJKwfo)
> 
> Hi Vikas, Thanks I know how to resolve but it's a little complex for this one

If possible can you share a screenshot or screen-recording of the conflicts?

**Comment by KendallDoesCoding** [comment]
> > > > 
> > > 
> > > 
> > > > How to resolve conflicts?
> > > 
> > > 
> > > He @KendallDoesCoding , The following resources may help you.
> > > 
> > > * GitHub Docs → [Resolving a merge conflict on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> > > * YouTube Video → [1.9: Resolving Merge Conflicts - Git and GitHub for Poets](https://youtu.be/JtIX3HJKwfo)
> > 
> > 
> > Hi Vikas, Thanks I know how to resolve but it's a little complex for this one
> 
> If possible can you share a screenshot or screen-recording of the conflicts?

image

**Comment by vikasganiga05** [comment]
Hey @KendallDoesCoding , Remove the line from the file. Which is marked red on the below screenshot :point_down: 

Screenshot

After removing those lines from the file you may see a button like `Mark as resolved` click it. Done :+1:

**Comment by KendallDoesCoding** [comment]
> Hey @KendallDoesCoding , Remove the line from the file. Which is marked red on the below screenshot 👇
> 
> Screenshot
> 
> After removing those lines from the file you may see a button like `Mark as resolved` click it. Done 👍 If you have any queries please let me know.

@vikasganiga05 Thanks a lot for the solution, will do this in the morning.

**Comment by vikasganiga05** [comment]
> > Hey @KendallDoesCoding , Remove the line from the file. Which is marked red on the below screenshot point_down
> > Screenshot
> > After removing those lines from the file you may see a button like `Mark as resolved` click it. Done +1 If you have any queries please let me know.
> 
> @vikasganiga05 Thanks a lot for the solution, will do this in the morning.

Most welcome :slightly_smiling_face:

**Comment by KendallDoesCoding** [comment]
A lot has been updated, will re-make the PR.

**Pull request closed without merging**